src/data/services/lms/urls.js

```diff
@@ -0,0 +1,63 @@
+import { StrictDict } from 'utils';
+import { configuration } from 'config';
+import { historyRecordLimit } from './constants';
+import { filterQuery, stringifyUrl } from './utils';
+
+const baseUrl = `${configuration.LMS_BASE_URL}`;
+
+const courseId = window.location.pathname.slice(1);
+
+const api = `${baseUrl}/api/`;
+const courses = `${baseUrl}courses/`;
```

Comment:
Missing a `/`? Alternatively, I don't think this is used so it could probably be deleted.
```suggestion
const courses = `${baseUrl}/courses/`;
```

Reply:
noted and agreed